astropy/cosmology/io/utils.py

```diff
@@ -0,0 +1,66 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+import numpy as np
+
+from astropy.cosmology.parameter import Parameter
+from astropy.table import Column
+from astropy.modeling import Parameter as ModelParameter
+
+
+def convert_Parameter_to_Column(parameter, value, meta=None):
+    """Convert a Cosmology Parameter to a Table Column.
+
+    Parameters
+    ----------
+    parameter : `astropy.cosmology.parameter.Parameter`
+    value : Any
+    meta : dict or None, optional
+        Information from the Cosmology's metadata.
+
+    Returns
+    -------
+    `astropy.table.Column`
+    """
+    format = None if value is None else parameter.format_spec
+    shape = (1,) + np.shape(value)  # minimum of 1d
+
+    col = Column(data=np.reshape(value, shape),
+                 name=parameter.name,
+                 dtype=None,  # inferred from the data
+                 shape=shape,
```

Comment:
`shape` and `length` are no longer needed since inferred from the `data`

Reply:
A nice simplification!